### PR TITLE
Add nop object

### LINKED ISSUE #760: `nop` object
We need to add `nop` object in `eo-runtime` which will encapsulate everything and do nothing. Also we need to add this object to papers.
Inspired by Assembler's `nop` (No Operations).

```
[args...] > nop
  TRUE > @
```

We can disable tests by this construction.
And lets' don't forget to fix `paper`.

### PR DESCRIPTION
**Pull request by kerelape**
Fix #760

### CODE REVIEW

File: eo-runtime/src/main/eo/org/eolang/nop.eo [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++package org.eolang
++rt jvm org.eolang:eo-runtime:0.0.0
```
Comment (Graur) [inline eo-runtime/src/main/eo/org/eolang/nop.eo:24]: @kerelape I think some meta information is missing here. Like `+version` and `+home`.

File: eo-runtime/src/test/eo/org/eolang/nop-tests.eo [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++package org.eolang
++junit
```
Comment (Graur) [inline eo-runtime/src/test/eo/org/eolang/nop-tests.eo:24]: @kerelape Same here. Some meta information is missing. Like `+version` and `+home`.

File: eo-runtime/src/test/eo/org/eolang/nop-tests.eo [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+[] > nop-is-true
+  nop > @
```
Comment (Graur) [inline eo-runtime/src/test/eo/org/eolang/nop-tests.eo:27]: @kerelape It would be great if you could use `eo-hamcrest` for tests cases

**Comment by Graur** [comment]
@kerelape Good one! Just a few comments above

**Review by Graur — APPROVED** [review]

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**